packages/toolkit/core-kit/src/regex.ts

```diff
@@ -4,6 +4,7 @@ export const phoneInputRegEx = /^\+?[\d-( )]+$/;
 export const usernameRegEx = /^[A-Z_a-z]\w*$/;
 export const webRedirectUriProtocolRegEx = /^https?:$/;
 export const mobileUriSchemeProtocolRegEx = /^[a-z][\d+_a-z-]*(\.[\d+_a-z-]+)+:$/;
+export const looseNativeUriSchemeProtocolRegEx = /^\S*:$/;
```

Comment:
Better to add some comments

```suggestion
/** ... */
export const looseNativeUriSchemeProtocolRegEx = /^\S*:$/;
```